go/porcelain/deploy.go

```diff
@@ -78,39 +77,25 @@ type uploadError struct {
 
 type FileBundle struct {
 	Name string
-	SHA  hash.Hash
+	Sum  string
 
+	// Path OR Buffer should be populated
+	Path   string
 	Buffer io.ReadSeeker
```

Comment:
might just not be easy to find in this PR - but in looking at the walk function, don't we not set the Buffer any more? Meaning Seek will fail as well as the write for functions?

Reply:
It is set for function uploads only since we create a zip archive in-memory.